provisionerd/runner/runner.go

```diff
@@ -1102,6 +1102,12 @@ func (r *Runner) runWorkspaceBuild(ctx context.Context) (*proto.CompletedJob, *p
 	}, nil
 }
 
+func (r *Runner) failedWorkspaceBuildf(format string, args ...interface{}) *proto.FailedJob {
```

Comment:
I feel like there must be a better way to do this...maybe `failedJobf().withWorkspaceBuildType()` or something but `proto.FailedJob` is generated so not sure how to add that.

Reply:
Meh, I think this is fine